helm_deploy/hmpps-micro-frontend-components/templates/services-cronjob.yaml

```diff
@@ -18,6 +18,8 @@ spec:
               env:
               - name: ENVIRONMENT_NAME
                 value: "{{ index .Values "generic-service" "env" "ENVIRONMENT_NAME" }}"
+              - name: ACTIVITIES_URL
+                value: "{{ index .Values "generic-service" "env" "ACTIVITIES_URL" }}"
```

Comment:
is there an entry for appointments needed here also or have i missed this? (may be answered already...)

Reply:
No. They used to have a double switch for a single application, 1 for activities and 1 for appointments, but decided to simplify it to just have to 1